docs/ops/sequence/AUGRUCell_dev.md

```diff
@@ -0,0 +1,144 @@
+# AUGRUCell  {#openvino_docs_ops_sequence_AUGRUCell_dev}
+
+**Versioned name**: *AUAUGRUCell-dev*
+
+**Category**: *Sequence processing*
+
+**Short description**: *AUGRUCell* represents a single AUGRU Cell (GRU with attentional update gate).
+
+**Detailed description**: The main difference between *AUGRUCell* and [GRUCell](./GRUCell_3.md) is the additional attention score input `A`, which is a multiplier for the update gate.
+The AUGRU formula is based on the [paper arXiv:1809.03672](https://arxiv.org/abs/1809.03672).
+
+```
+AUGRU formula:
+  *  - matrix multiplication
+ (.) - Hadamard product (element-wise)
+
+ f, g - activation functions
+ z - update gate, r - reset gate, h - hidden gate
+ a - attention score
+
+  rt = f(Xt*(Wr^T) + Ht-1*(Rr^T) + Wbr + Rbr)
+  zt = f(Xt*(Wz^T) + Ht-1*(Rz^T) + Wbz + Rbz)
+  ht = g(Xt*(Wh^T) + (rt (.) Ht-1)*(Rh^T) + Rbh + Wbh)  # 'linear_before_reset' is False
+
+  zt' = (1 - at) (.) zt  # multiplication by attention score
+
+  Ht = (1 - zt') (.) ht + zt' (.) Ht-1
+```
+
+Activation functions for gates: *sigmoid* for f, *tanh* for g.
+
+**Attributes**
+
+* *hidden_size*
+
+  * **Description**: *hidden_size* specifies hidden state size.
+  * **Range of values**: a positive integer
+  * **Type**: `int`
+  * **Required**: *yes*
+
+
+The rest of the attributes are supported only with default values:
+
+* *activations*
+
+  * **Description**: activation functions for gates
+  * **Range of values**: any combination of *relu*, *sigmoid*, *tanh*
```

Comment:
Didn't we agree to support just the single value (default one) for all attributes? Or did we decide to support all?

Reply:
The cell has sigmoid and tanh.

Reply:
We seemed to agree to omit these values until we get real models with it. List of these values requires us to implement this support in reference

Reply:
We agreed, I just had this idea with "Supported values:" section (reverted).
Updated to just: *sigmoid*, *tanh*